Apps/Sandcastle/gallery/GeoJSON simplestyle.html

```diff
@@ -27,17 +27,23 @@
 function startup(Cesium) {
     "use strict";
 //Sandcastle_Begin
-var viewer = new Cesium.Viewer('cesiumContainer', {timeline : false, animation : false});
-
 //Load a GeoJSON file containing simplestyle information.
 //To learn more about simplestyle, see https://github.com/mapbox/simplestyle-spec
-viewer.dataSources.add(Cesium.GeoJsonDataSource.fromUrl('../../SampleData/simplestyles.geojson'));
-
-//Set the camera to view the data.
-viewer.camera.viewRectangle(Cesium.Rectangle.fromDegrees(-0.1, 0.2, 1.5, -1.1));
 
 //In this particular example, the name of each entity is set to its maki icon identifier.
 //Clicking on each billboard will show it's identifier in the InfoBox.
+
+var viewer = new Cesium.Viewer('cesiumContainer', {
+    sceneMode : Cesium.SceneMode.SCENE2D,
+    timeline : false,
+    animation : false
+});
+
+var dataSource = new Cesium.GeoJsonDataSource();
+viewer.dataSources.add(dataSource);
+dataSource.loadUrl('../../SampleData/simplestyles.geojson').then(function(){
```

Comment:
In order to make this a little easier, I updated the `zoomTo` and `flyTo` methods so that if the data source is in the process of loading, it waits until it's done before doing the zoom (these methods were already asynchronous).  This turns bring it down to 3 lines instead of 6 and eliminates the pitfall with many users who try and use the data source before it's ready.

```
var dataSource = Cesium.GeoJsonDataSource.fromUrl('../../SampleData/simplestyles.geojson');
viewer.dataSources.add(dataSource);
viewer.zoomTo(dataSource);
```

Obviously we'll continue to look for ways to improve the API when we get more user feedback.